src/utils.ts

```diff
@@ -0,0 +1,10 @@
+export function merge(target: object, ...srcs: object[]): object {
```

Comment:
Why not use `Object.assign` because you need to use polyfill?

Reply:
Sure, that's the reason. As we're targeting IE9+

Reply:
@WoodNeck 

Create object.assign polyfill in tsc.